python/pyspark/sql/functions.py

```diff
@@ -44,6 +44,20 @@ def _(col):
     return _
 
 
+def _create_zero_arg_function(name, doc=""):
```

Comment:
this is actually not zero arg is it? 

I'm tempted to not rely on this magic function here, since we only have two functions. Then you can also have docstring test inline.